src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Automatic Instrumentation Preview
+sidebar_order: 10
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+This is a preview API, available in `7.2.0-beta.0` and above. Features available in a preview API are still in progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+
+</Note>
+
+## UIViewController Instrumentation
+
+This feature is available for iOS, tvOS, and Mac Catalyst, and only works for UIViewControllers.
+
+The UIViewController Instrumentation, once enabled, captures transactions whenever your app loads a ViewController. The SDK sets the transaction name to the name of the ViewController, including the module; for example, `Your_App.MainViewController`, and the transaction operation to `ui.load`. The SDK creates spans to provide insight into the length of time each consumed by each of the following methods.
+
+![UIViewController Transaction](ui-view-controller-transaction.png)
```

Comment:
`h`:  Update these screenshots as the operation names changed after merging https://github.com/getsentry/sentry-cocoa/pull/1164